src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -79,7 +102,7 @@ const NavHeader = (props) => {
           ? dimen.size.TOPIC_FEED_HEADER_HEIGHT
           : dimen.size.TOPIC_FEED_HEADER_HEIGHT + normalize(4);
     }
-    return Math.round((containerHeight - domainHeight) / 2);
+    return Math.round((containerHeight - domainHeight) / 2) + searchHeight;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The calculation for the return value now includes `searchHeight`. Ensure that this value is always defined and is a number to avoid potential NaN results.



<!-- This is an auto-generated comment by OSS CodeRabbit -->